docs/modules/ROOT/pages/deploy-management-center.adoc

```diff
@@ -0,0 +1,210 @@
+= Deploy Management Center with the Hazelcast Platform Operator for Kubernetes
+:description: In this tutorial, you'll deploy an instance of Management Center using Hazelcast Platform Operator for Kubernetes.
+
+{description}
+
+== Before you Begin
+
+You need a Kubernetes or Openshift cluster, and the `kubectl` or `oc` command-line tool must be configured to communicate with your cluster.
+
+Please make sure xref:get-started.adoc[Hazelcast Platform Operator for Kubernetes] is up and running.
+
+== Deploy Management Center
+
+You can monitor the Hazelcast cluster by starting Management Center.
+
+[tabs]
+====
+Open Source::
++
+--
+
+. Create the `ManagementCenter` custom resource file and name it `management-center.yaml`.
++
+[source,yaml,subs="attributes+"]
+----
+include::ROOT:example$/management-center.yaml[]
+----
+
+. Apply it with the following command to start Management Center.
++
+.For Kubernetes
+[source,shell]
+----
+kubectl apply -f management-center.yaml
```

Comment:
Since we add sample yaml for each feature within MC, could you simplify the existing sample yamls? I checked them and for example, they still have the persistence section. I think we don't need to show it in all the sample yamls like `management-center.yaml`, `management-center-with-licence-key.yaml`.